ansible/roles/automation/nightly_pr/defaults/main.yml

```diff
@@ -30,6 +30,13 @@ nightly_jobs:
     flow: "ci"
     branch: "ipa-4-7"
     prci_config: "nightly_master.yaml"
+  - name: testing_ipa-4.6
+    weekdays: "7"
+    hour: "10"
+    minute: "00"
+    flow: "ci"
+    branch: "ipa-4-6"
+    prci_config: "nightly_ipa-4-6.yaml"
```

Comment:
is this in prci_definitions?

Reply:
Yes: https://github.com/freeipa/freeipa/blob/ipa-4-6/ipatests/prci_definitions/nightly_ipa-4-6.yaml
Just the same as used by previous runs: https://github.com/freeipa/freeipa/pull/3283/files